README.md

```diff
@@ -1658,7 +1668,8 @@ manager, your future manager, or a recruiter, or some combination) multiple time
 signing. This helps you negotiate details and offers you a chance to get to know the
 people, the company, and the role, so that you can make the best decision for you.
 
-Things to look for in the offer letter include:
+Things to look for in [the offer letter](https://blog.carta.com/a-better-offer-letter/)
```

Comment:
Hm. This is an interesting blog post likely worth linking to, but it's just Carta's way of doing it, not what's typical. Seems like this isn't a clear context around the link.

Reply:
Agreed. I think it is helpful to offer a specific example, though! Just change the link text to make it clear this is how one company, Carta, does it. "Here's **one example from Carta** of what's included in an offer letter. Think you can expect to see include: ...."

Reply:
The Carta thing is nice, but just not typical (its intent is also marketing for them).

This is closer to what we need. Let's use it instead unless we can find a better one. https://www.glassdoor.com/blog/how-to-read-offer-letter/

Reply:
I removed the Carta link and replaced with Glassdoor. Will see if there better links out there and consider further but wanted to get fix in.